dagger/dag_creator/airflow/operator_creators/spark_creator.py

```diff
@@ -64,8 +64,7 @@ def _create_operator(self, **kwargs):
                 job_args=self._generate_command(),
                 spark_args=self._generate_spark_args(),
                 s3_files_bucket=self._task.s3_files_bucket,
-                extra_py_files=self._task.extra_py_files,
-                emr_master=self._task.emr_master,
```

Comment:
Do we want to completely remove this or support both options?

Reply:
We can but do you see a use case for it ?